_templates/release.qmd

```diff
@@ -10,3 +10,20 @@ We are very excited to announce the release of a new {{ pkgname }} version {{ v
 Here is an automatically generated summary of the changes in this version.
 
 {{ notes }}
+
+## Acknowledgements
+
+```{r ctbs, results='asis', echo=FALSE, message = FALSE}
+releases <- gh::gh("GET /repos/{owner}/{repo}/releases", 
+               owner = "epiverse-trace",
+               repo = "{{ pkgname }}") |>
+  purrr::discard(~ grepl("\\d+\\.\\d+\\.[1-9]\\d*$", .x$tag_name))
+
+ctbs <- usethis::use_tidy_thanks(
+  paste0('epiverse-trace/', 
+    "{{ pkgname }}"), 
+  from = releases[[length(releases) - 1]][["tag_name"]], 
```

Comment:
```suggestion
  from = releases[[2]][["tag_name"]], 
```

Reply:
👍 I thought I changed this but apparently overlooked this 😅